<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>espeak_preferences</class>
 <widget class="QWidget" name="espeak_preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="10" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="use_bark">
     <property name="title">
      <string>SPD</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="5" column="0">
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Pitch</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="SliderWidget" name="pitch" native="true">
           <property name="slider_minimum" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_maximum" stdset="0">
            <number>100</number>
           </property>
           <property name="spinbox_minimum" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="spinbox_maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="display_as_float" stdset="0">
            <bool>true</bool>
           </property>
           <property name="settings_property" stdset="0">
            <string>pitch</string>
           </property>
           <property name="slider_single_step" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_page_step" stdset="0">
            <number>10</number>
           </property>
           <property name="spinbox_single_step" stdset="0">
            <double>0.010000000000000</double>
           </property>
           <property name="spinbox_page_step" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="slider_callback" stdset="0">
            <string>handle_value_change</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Gender</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QComboBox" name="gender_combobox">
           <item>
            <property name="text">
             <string>Male</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Female</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Voice</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QComboBox" name="voice_combobox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Robotic, fast, no VRAM usage</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string>Volume</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="SliderWidget" name="volume" native="true">
           <property name="slider_minimum" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_maximum" stdset="0">
            <number>100</number>
           </property>
           <property name="spinbox_minimum" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="spinbox_maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="display_as_float" stdset="0">
            <bool>true</bool>
           </property>
           <property name="settings_property" stdset="0">
            <string>volume</string>
           </property>
           <property name="slider_single_step" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_page_step" stdset="0">
            <number>10</number>
           </property>
           <property name="spinbox_single_step" stdset="0">
            <double>0.010000000000000</double>
           </property>
           <property name="spinbox_page_step" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="slider_callback" stdset="0">
            <string>handle_value_change</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Rate</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="SliderWidget" name="rate" native="true">
           <property name="slider_minimum" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_maximum" stdset="0">
            <number>100</number>
           </property>
           <property name="spinbox_minimum" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="spinbox_maximum" stdset="0">
            <double>1.000000000000000</double>
           </property>
           <property name="display_as_float" stdset="0">
            <bool>true</bool>
           </property>
           <property name="settings_property" stdset="0">
            <string>rate</string>
           </property>
           <property name="slider_single_step" stdset="0">
            <number>1</number>
           </property>
           <property name="slider_page_step" stdset="0">
            <number>10</number>
           </property>
           <property name="spinbox_single_step" stdset="0">
            <double>0.010000000000000</double>
           </property>
           <property name="spinbox_page_step" stdset="0">
            <double>0.100000000000000</double>
           </property>
           <property name="slider_callback" stdset="0">
            <string>handle_value_change</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Language</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="0">
          <widget class="QComboBox" name="language_combobox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Override the system language</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWidget</class>
   <extends>QWidget</extends>
   <header>airunner/widgets/slider/slider_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>language_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>espeak_preferences</receiver>
   <slot>language_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>voice_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>espeak_preferences</receiver>
   <slot>voice_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gender_combobox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>espeak_preferences</receiver>
   <slot>gender_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>language_changed(QString)</slot>
  <slot>gender_changed(QString)</slot>
  <slot>voice_changed(QString)</slot>
  <slot>use_bark_changed(bool)</slot>
  <slot>enable_tts_changed(bool)</slot>
  <slot>model_changed(QString)</slot>
 </slots>
</ui>
